aws-fargate-cfn/op-scim-bridge.yaml

```diff
@@ -0,0 +1,392 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  Deploys 1Password SCIM bridge using Amazon ECS and AWS Fargate. Provisions and
+  configures a VPC, 2 public subnets, an internet gateway, a route table, an
+  ALB, an ACM certificate, Route 53 DNS records, an AWS secret to store
+  credentials for your SCIM bridge, security groups, and required IAM roles.
+Metadata: 
+  AWS::CloudFormation::Interface: 
+    ParameterGroups:
+      - 
+        Label: 
+          default: ""
+        Parameters: 
+          - VPCCIDR
+          - Route53HostedZone
+          - DomainName
+          - SCIMBridgeVersion
+          - scimsession
+    ParameterLabels: 
+      VPCCIDR: 
+        default: "VPC CIDR"
+      Route53HostedZone:
+        default: "Route 53 hosted zone"
+      DomainName:
+        default: "1Password SCIM bridge domain name"
+      SCIMBridgeVersion:
+        default: "1Password SCIM bridge version"
+Parameters:
+  VPCCIDR:
+    Type: String
+    Default: 10.0.0.0/16
+    Description: A CIDR block for the VPC to be created
+  Route53HostedZone:
+    Type: 'AWS::Route53::HostedZone::Id'
+    Description: >-
+      The Route 53 hosted zone in which to create DNS records for ACM validation and the ALB
+  DomainName:
+    Type: String
+    Default: scim.example.com
+    Description: >-
+      A fully qualified domain name for your SCIM bridge; this must be in the
+      domain of the selected Route 53 hosted zone (where the record will be
+      created)
+  scimsession:
+    Type: String
+    Description: >-
+      The plain text contents of the scimsession file created during the
+      automated user provisioning setup in your 1Password account
+    NoEcho: true
+  SCIMBridgeVersion:
+    Type: String
+    Default: 'v2.8.1'
```

Comment:
We should drop the "v" from the parameter and account for this in line 83 Image: !Sub '1password/scim:v${SCIMBridgeVersion}'

This caused confusion when I attempted to run this from CLI where the defaults are not shown in the for each parameter.